src/wmtk/simplex/cofaces_single_dimension.cpp

```diff
@@ -33,32 +56,24 @@ std::vector<Tuple> cofaces_single_dimension_tuples(
     return {};
 }
 
-std::vector<Tuple> cofaces_single_dimension_tuples(
-    const TriMesh& mesh,
-    const Simplex& my_simplex,
+std::vector<Simplex> cofaces_single_dimension_simplices(
```

Comment:
This is a function that is going to be used by everyone. It should have 100% coverage.